optaplanner-examples/pom.xml

```diff
@@ -175,6 +175,38 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>javax.json</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.mock-server</groupId>
+      <artifactId>mockserver-netty</artifactId>
+    </dependency>
   </dependencies>
 
+  <dependencyManagement>
```

Comment:
move dependencyManagement to the optaplanner root pom, we'll work in a separate PR/issue to move it from there to where it needs to be.